ReflectometryServer/pv_wrapper.py

```diff
@@ -38,6 +42,15 @@ def __init__(self, base_pv, ca=None):
         self._after_sp_change_listeners = set()
         self._after_is_changing_change_listeners = set()
 
+        if min_velocity_scale_factor is None:
+            self._min_velocity_scale_factor = DEFAULT_SCALE_FACTOR
+        if min_velocity_scale_factor < 0:
```

Comment:
Maybe it should be `min_velocity_scale_factor <= 0:` or else minimum velocity could end up becoming 0?

Reply:
Yes well spotted, it's actually worse than that (it would cause a division by zero exception)